modules/subscriptions/views.php

```diff
@@ -322,18 +338,34 @@ static function render_widget_subscription_form( $args, $instance, $subscribe_em
 						echo wpautop( $subscribe_text );
 					}
 					if ( $show_subscribers_total && $subscribers_total ) {
-						/* translators: %s: number of folks following the blog */
-						echo wpautop( sprintf( _n( 'Join %s other follower', 'Join %s other followers', $subscribers_total ), number_format_i18n( $subscribers_total ) ) );
+						?>
+						<div class="jetpack-subscribe-count">
+							<p>
+							<?php
+							/* translators: %s: number of folks following the blog */
+							echo esc_html( sprintf( _n( 'Join %s other follower', 'Join %s other followers', $subscribers_total, 'jetpack' ), number_format_i18n( $subscribers_total ) ) );
+							?>
+							</p>
+						</div>
+						<?php
 					}
 					?>
-                    <p><input type="text" name="email" style="width: 95%; padding: 1px 10px"
-                              placeholder="<?php esc_attr_e( 'Enter your email address' ); ?>" value=""
-                              id="subscribe-field<?php if ( Jetpack_Subscriptions_Widget::$instance_count > 1 ) {
-						          echo '-' . Jetpack_Subscriptions_Widget::$instance_count;
-					          } ?>"/></p>
+					<p id="subscribe-email">
+                    	<input type="text" name="email"
+                    		<?php if ( ! empty( $email_field_classes ) ) { ?>
+	                        	class="<?php echo esc_attr( $email_field_classes ); ?>"
+	                    	<?php } ?>
+		                	<?php if ( ! empty( $email_field_styles ) ) { ?>
+			                	style="<?php echo esc_attr( $email_field_styles ); ?>"
+		                	<?php } else { ?>
+                    			style="width: 95%; padding: 1px 2px"
+                    		<?php } ?> placeholder="<?php esc_attr_e( 'Enter your email address' ); ?>" value="" id="subscribe-field<?php if ( Jetpack_Subscriptions_Widget::$instance_count > 1 ) {
+								echo '-' . Jetpack_Subscriptions_Widget::$instance_count;
+					    	} ?>"/>
+					</p>
```

Comment:
I wonder if we should take that opportunity to make all this a bit more readable, here and below? I haven't tested, but something like this?

```diff
diff --git a/modules/subscriptions/views.php b/modules/subscriptions/views.php
index 56dbd786f..c6cfe77ab 100644
--- a/modules/subscriptions/views.php
+++ b/modules/subscriptions/views.php
@@ -351,17 +351,32 @@ class Jetpack_Subscriptions_Widget extends WP_Widget {
 					}
 					?>
 					<p id="subscribe-email">
-                    	<input type="text" name="email"
-                    		<?php if ( ! empty( $email_field_classes ) ) { ?>
-	                        	class="<?php echo esc_attr( $email_field_classes ); ?>"
-	                    	<?php } ?>
-		                	<?php if ( ! empty( $email_field_styles ) ) { ?>
-			                	style="<?php echo esc_attr( $email_field_styles ); ?>"
-		                	<?php } else { ?>
-                    			style="width: 95%; padding: 1px 2px"
-                    		<?php } ?> placeholder="<?php esc_attr_e( 'Enter your email address' ); ?>" value="" id="subscribe-field<?php if ( Jetpack_Subscriptions_Widget::$instance_count > 1 ) {
-								echo '-' . Jetpack_Subscriptions_Widget::$instance_count;
-					    	} ?>"/>
+						<?php
+						printf(
+							'<input
+								type="text"
+								name="email"
+								%1$s
+								style="%2$s"
+								placeholder="%3$s"
+								value=""
+								id="subscribe-field%4$s
+							/>',
+							( ! empty( $email_field_classes )
+								? 'class="' . esc_attr( $email_field_classes ) . '"'
+								: ''
+							),
+							( ! empty( $email_field_styles )
+								? esc_attr( $email_field_styles )
+								: 'width: 95%; padding: 1px 2px'
+							),
+							esc_attr__( 'Enter your email address' ), // phpcs:ignore WordPress.WP.I18n.MissingArgDomain -- This is only used on WordPress.com.
+							( absint( self::$instance_count ) > 1
+								? '-' . absint( self::$instance_count )
+								: ''
+							)
+						);
+						?>
 					</p>
 				<?php endif; ?>
 

```